math/mathcore/inc/Fit/Fitter.h

```diff
@@ -521,7 +521,8 @@ class Fitter {
    /// return pointer to used objective function
    /// if using an external one use external, otherwise use the one stored
    /// in shared ptr
-   const ROOT::Math::IMultiGenFunction * ObjFunction() const {
+   const ROOT::Math::IBaseFunctionMultiDimTempl<double> * ObjFunction() const {
+   //const ROOT::Math::IMultiGenFunction * ObjFunction() const {
```

Comment:
Thanks for working on this, @lmoneta!  Is this worth keeping as a comment? -- I would delete otherwise.